plugins/shrs_completion/src/setup.rs

```diff
@@ -0,0 +1,118 @@
+use rhai::{Array, CustomType, Dynamic, Engine, ImmutableString, Scope, Shared, TypeBuilder};
+use shrs::prelude::*;
+fn default_completion(completion: String) -> Completion {
```

Comment:
try to add a newline between functions and also the imports, makes it look a bit more clean